src/jsonpointer/include/sourcemeta/jsontoolkit/jsonpointer.h

```diff
@@ -233,6 +245,12 @@ auto stringify(const Pointer &pointer,
                std::basic_ostream<JSON::Char, JSON::CharTraits> &stream)
     -> void;
 
+// TODO: add doc
+SOURCEMETA_JSONTOOLKIT_JSONPOINTER_EXPORT
+auto stringify(const WeakPointer &pointer,
+               std::basic_ostream<JSON::Char, JSON::CharTraits> &stream)
+    -> void;
+
```

Comment:
I had linker issues trying to "templating" these methods. I could give it a new shot in a another PR.

Reply:
I like the current approach of adding proper non-templated overloads here. Then on the `.cc` file, they can share an internal templated helper. Maybe worth extracting these already into a separate PR?